evaluation_experiments/EXP3_cibersort_eval.ipynb

```diff
@@ -0,0 +1,6160 @@
+{
```

Comment:
<p>I don't fully understand this equation. What is (ref_bp, scaled)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pred_missing_celltypes/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>